services/response_code.proto

```diff
@@ -1328,4 +1328,9 @@ enum ResponseCodeEnum {
    * The range provided in RandomGenerate transaction is negative.
    */
   INVALID_RANDOM_GENERATE_RANGE = 324;
+
+  /**
+ * The full prefix signature for precompile is not valid
+ */
+  INVALID_FULL_PREFIX_SIGNATURE_FOR_PRECOMPILE = 325;
```

Comment:
Actually need to merge from main as this ordinal is already taken.

Reply:
+1